src/fqs/test_fqs.py

```diff
@@ -0,0 +1,303 @@
+from typing import Any, Callable, FrozenSet, Set
+
```

Comment:
You mentioned you wanted to add authorship info?

Reply:
Resolved in [4b94568](https://github.com/Geopipe/fqs/pull/1/commits/4b94568e86e1d7ca1f66a29dffb5a0a867b8bb95)